packages/services/src/services/KalturaPlayer/KalturaPlayer.js

```diff
@@ -177,7 +177,10 @@ class KalturaPlayerAPI {
    *
    * @param {string} mediaId  The mediaId we're embedding the placeholder for.
    * @param {string} targetId The targetId the ID where we're putting the placeholder.
-   * @param {boolean} flashvars Determine any extra param or plugin for the player.
+   * @param {object} flashvars Determine any extra param or plugin for the player.
+   * @param {boolean} useIbmMetrics Whether or not should IBM Metrics events be fired.
+   * @param {Function} customReadyCallback Determine any extra functions that should be executed
```

Comment:
Minor thing, but can add a note that the player object is passed in the custom method in order to add additional custom functionality (e.g. listeners, etc). Is there a link we can send users to for a full API that they can tap into?